pyrightconfig.json

```diff
@@ -0,0 +1,3 @@
+{
+    "reportFunctionMemberAccess": false
+}
```

Comment:
I've backported this in #22, so it can be merged before the rest of this PR, since we're already sometimes running `pyright` locally. In the likely event that #22 is merged first, this PR can be rebased onto main, eliminating this part of the diff by integrating the change. In the much less likely event that this is merged first, #22 could just be closed without merging.